libpod/container_api.go

```diff
@@ -642,7 +644,7 @@ func (c *Container) WaitForExit(ctx context.Context, pollInterval time.Duration)
 					return true, 0, nil
 				}
 			}
-			return true, -1, fmt.Errorf("%w (container in state %s)", err, c.state.State)
+			return true, -1, fmt.Errorf("%w (container state: %s, timed out: %v)", err, c.state.State, timedout)
```

Comment:
I don't have a sense for how this will help... but one possibly useless suggestion: instead of a true/false, WDYT of a string value instead, invisible by default?
```
    timedout := ""
   ...
       timedout = " [timed out waiting for conmon]"
   ...
        ..."(container in state %s)%s"
```

Reply:
Pushed.  I used a different message though (i.e., the one from the debug log) as we're not waiting for conmon at this point but for the container to exit _after_ conmon has exited.